app/client/cli/debug.go

```diff
@@ -66,17 +70,65 @@ func NewDebugSubCommands() []*cobra.Command {
 	return commands
 }
 
-// NewDebugCommand returns the cobra CLI for the Debug command.
-func NewDebugCommand() *cobra.Command {
+// newDebugUICommand returns the cobra CLI for the Debug UI interface.
+func newDebugUICommand() *cobra.Command {
 	return &cobra.Command{
-		Use:               "debug",
-		Short:             "Debug utility for rapid development",
+		Use:               "debug_ui",
```

Comment:
Nit: I'm generally not a fan of underscores in CLI command names. Consider removing the underscore. Sidenote: This command is used often enough it is probably worth adding an alias to `d` or something.

Reply:
[`43ef141` (#902)](https://github.com/pokt-network/pocket/pull/902/commits/43ef141122b4e7385702596f068d4dc40372cd2b)

Reply:
Lol @ `dui`. It's perfect, let's never change it.

Reply:
Just`dui`t